sigstore-java/src/main/java/dev/sigstore/bundle/BundleFactory.java

```diff
@@ -38,7 +36,16 @@ public class BundleFactory {
   public static String createBundle(KeylessSigningResult signingResult) {
     Bundle bundle = BundleFactoryInternal.createBundleBuilder(signingResult).build();
     try {
-      return JSON_PRINTER.print(bundle);
+      String jsonBundle = BundleFactoryInternal.JSON_PRINTER.print(bundle);
```

Comment:
I moved the static field to a different class so protobuf does not leak to the public API